index.html

```diff
@@ -192,7 +192,7 @@ <h2 class="p-muted-heading u-align-text--center">Who&rsquo;s using Vanilla</h2>
   <div class="row">
     <div class="col-3">
       <h2 class="p-heading--three">Contribute</h2>
-      <p><a class="github-button" href="https://github.com/vanilla-framework" data-size="large" data-show-count="true" aria-label="Follow @vanilla-framework on GitHub">Follow @vanilla-framework</a></p>
+      <p><a class="github-button" href="https://github.com/canonical-web-and-design/vanilla-framework" data-size="large" data-show-count="true" aria-label="Follow @vanilla-framework on GitHub">Follow @vanilla-framework</a></p>
       <p><a class="github-button" href="https://github.com/vanilla-framework/vanilla-framework" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star vanilla-framework/vanilla-framework on GitHub">Star</a></p>
       <a class="github-button" href="https://github.com/vanilla-framework/vanilla-framework/fork" data-icon="octicon-repo-forked" data-size="large" data-show-count="true" aria-label="Fork vanilla-framework/vanilla-framework on GitHub">Fork</a>
```

Comment:
Oh, the fork button has old link as well.

Reply:
Will update 👍